### ISSUE TITLE
Konsole: Vim visual line invisible

### ISSUE THREAD

**Issue by ponchofiesta** [sonph/onehalf#56]
Using the light Konsole theme causes an invisible "visual line" marking in Vim.

Reproduce:
- Open any text file in Vim
- Press Shift+V
- Go down using the cursor

You should the the marked lines but you see nothing.

**Comment by sonph** [comment]
Do you mind attaching a screenshot? Are you using the light theme or dark theme?

I don’t use Konsole so it’s kinda hard for me to debug this. The color you should see is `s:selection` as defined in the light/dark color theme files. [Example](https://github.com/sonph/onehalf/blob/master/vim/colors/onehalfdark.vim#L39).

Are you using a version of vim that supports true colors? Does Konsole support true colors?

**Comment by ponchofiesta** [comment]
Yes, the light theme. I already found the cause. Color 8 (Farbe 8) is white (or missing?) and looks like the background color. Setting it to another color makes the visual line visible - grey here:

![image](https://user-images.githubusercontent.com/12248938/60324750-e862cd00-9985-11e9-897f-7d2dc7abb2ea.png)
